universe/Building.h

```diff
@@ -61,13 +61,7 @@ class FO_COMMON_API Building : public UniverseObject {
 protected:
     friend class Universe;
     /** \name Structors */ //@{
-    Building() :
-        UniverseObject(),
-        m_building_type(),
-        m_planet_id(INVALID_OBJECT_ID),
-        m_ordered_scrapped(false),
-        m_produced_by_empire_id(ALL_EMPIRES)
-    {}
+    Building() {}
```

Comment:
I don't see the style conflict. (What should I be looking at?)
The public destructor a few lines down has the {} on the same line.